app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_count)
+  end
```

Comment:
**OPTIONAL**

- [ ] Since you are using `after_save: update_likes_counter` it is usually good practice to make the method itself private.